inc/class-wpseo-validator.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Internals
+ */
+
+/**
+ * Class WPSEO_Validator
+ */
+class WPSEO_Validator {
+
+	/**
+	 * Validates whether the passed variable is a boolean.
+	 *
+	 * @param mixed $variable The variable to validate.
+	 *
+	 * @return bool Whether or not the passed variable is a valid boolean.
+	 */
+	public static function is_boolean( $variable ) {
+		return is_bool( $variable ) !== null;
```

Comment:
This always results in true, because `is_bool` will return `TRUE` or `FALSE` but never `NULL`.